ci/jenkins/jobs/macros.yaml

```diff
@@ -263,3 +263,42 @@
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           chmod +x ./ci/jenkins/test-rancher.sh
           ./ci/jenkins/test-rancher.sh --cluster-name rancher-test --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}}
+
+- builder:
```

Comment:
Can prefer to combine two builders in one and then call by passing test _type conformance, e2e and network policy
builders:
  - <<: *builder-kind-ipv6
    name: builder-kind-ipv6-ds-conformance
    env:
      test_type: conformance

  - <<: *builder-kind-ipv6
    name: builder-kind-ipv6-ds-e2e
    env:
      test_type: e2e
and use a common builder name as all other commands are same except test name
builders:
  - name: builder-kind-ipv6

Reply:
Hi, I tried it, I think we can't combine both the builder into one because when we run conformance test then we pass variable with double curly brackets but in e2e we use single curly bracket.

Reply:
You can maintain two different builders, but the reason is not of curly brackets, even though they are two and one, because {} it is just as an escape variable and is used to differentiate with the jenkins variable.